contributing/development/core_and_modules/internal_rendering_architecture.rst

```diff
@@ -228,8 +228,8 @@ support Vulkan. OpenGL 3.3 Core Profile is used on desktop platforms to run this
 driver, as most graphics drivers on desktop don't support OpenGL ES.
 WebGL 2.0 is used for web exports.
 
-It is possible to use the use of OpenGL ES 3.0 directly on desktop platforms
-using the ``--rendering-driver opengl3_es`` command line argument, although this
+It is possible to use OpenGL ES 3.0 directly on desktop platforms
```

Comment:
I think it's only supposed on Linux, not on all desktop platforms. Windows and macOS can use GLES over Angle, but not directly.

Reply:
Indeed, Linux drivers have support for it but Windows/macOS drivers don't.